examples/prom/mixed_nonlinear_diffusion.cpp

```diff
@@ -619,8 +618,8 @@ int main(int argc, char *argv[])
                    "Enable or disable the online phase.");
     args.AddOption(&merge, "-merge", "--merge", "-no-merge", "--no-merge",
                    "Enable or disable the merge phase.");
-    args.AddOption(&use_sopt, "-sopt", "--sopt", "-no-sopt", "--no-sopt",
-                   "Use S-OPT sampling instead of DEIM for the hyperreduction.");
+    args.AddOption(&samplingType, "-hrtype", "--hrsamplingtype",
+                   "Sampling type for hyperreduction.");
```

Comment:
I recommend adding different hyper-reduction sampling types in the regression test for this example, similar to the mixed nonlinear diffusion.

Reply:
Done.